kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/zte/me3630/ZteMe3630.java

```diff
@@ -0,0 +1,368 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eurotech and/or its affiliates
```

Comment:
I believe that this needs to be changed to something like:
```
/*******************************************************************************
  * Copyright (c) 2019 3 PORT d.o.o.
```
since it's a file you contributed.